### PR TITLE
Revert d46a0b8 and add fix for sentry tests

### PR DESCRIPTION
**Pull request by onewland**
Original revert PR https://github.com/getsentry/snuba/pull/2563

### CODE REVIEW

**Comment by volokluev** [comment]
How does this fix the sentry tests?

**Comment by onewland** [comment]
> How does this fix the sentry tests?

[This line](https://github.com/getsentry/snuba/pull/2564/files#diff-1177fdef1146fb34687b873f0f151ba90d542fe92e061aafb2b84d0b918b78c7R42) is the relevant change.

The sentry metrics-related tests use a hardcoded date of roughly 1 day ago: [source](https://github.com/getsentry/sentry/blob/7ae9b36b26f455d35bc67b8f9d85275f44d9d4ab/tests/snuba/api/endpoints/test_organization_sessions.py#L24-L30). I refactored these from a constant date to a dynamic one because we have a 90-day TTL on these tables, and this is pulling from ClickHouse. The change in this PR has us reading from a new table but that migration has us only writing to the new table from after 3-29-22 00:00:00 UTC. So the sentry tests are reading from a table where this data flow is not enabled yet.

**Comment by onewland** [comment]
Looks like tests are still failing, investigating

File: snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py:42]: Why not just removing it. 
Also I think you will have the same issue with sets

Comment (lynnagara) [inline snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py:42]: Separately to the tests, can you explain what this date actually means? Like, what is different before and after this date?

**Comment by fpacifici** [comment]
I am not sure this will fix the problem though. 
Since the migration already landed in master and it was applied on everybody's developer environment, changing the timestamp will not have any impact for those that already applied the migration. To fix that you will have to add a new migration that creates a new matview and drops the existing.

File: snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py
Comment (onewland) [reply snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py]: > I am not sure this will fix the problem though. Since the migration already landed in master and it was applied on everybody's developer environment, changing the timestamp will not have any impact for those that already applied the migration. To fix that you will have to add a new migration that creates a new matview and drops the existing.

yeah, I'm doing this now

Comment (onewland) [reply snuba/migrations/snuba_migrations/metrics/0030_metrics_distributions_v2_writing_mv.py]: It was just a random marker so that Filippo could only backfill to that point and we know any row with a timestamp after that comes from the materialized view

**Comment by onewland** [comment]
Please wait until I explicitly request a review before adding more comments.

**Comment by onewland** [comment]
Ready for review again

File: snuba/migrations/snuba_migrations/metrics/0032_redo_0030_and_0031_without_timestamps.py
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/metrics/0032_redo_0030_and_0031_without_timestamps.py:50]: You should not need this. Please remove

File: snuba/migrations/snuba_migrations/metrics/0032_redo_0030_and_0031_without_timestamps.py
Comment (fpacifici) [inline snuba/migrations/snuba_migrations/metrics/0032_redo_0030_and_0031_without_timestamps.py:66]: Same, please remove

**Review by fpacifici — APPROVED** [review]
Please fix the conditions before merging

**Pull request merged**
